controllers/fakeProgressOutput.sh

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
What's with the whitespace and indentation?

Reply:
Not sure what happened there.